roles/ceph-mgr/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: restart prometheus module
+  delegate_to: "{{ groups[mon_group_name][0] }}"
+  block:
+    - name: disable prometheus module
+      command: "{{ container_exec_cmd_mgr | default('') }} ceph --cluster {{ cluster }} mgr module disable prometheus"
+    - name: enable prometheus module
+      command: "{{ container_exec_cmd_mgr | default('') }} ceph --cluster {{ cluster }} mgr module enable prometheus"
```

Comment:
```suggestion
      command: "{{ container_exec_cmd_mgr | default('') }} ceph --cluster {{ cluster }} mgr module enable prometheus"
 
```

Reply:
@guits maybe something went wrong, because you suggested exactly same line